README.md

```diff
@@ -79,7 +78,55 @@ IBM Watson™ Compare and Comply is discontinued. Existing instances are support
 
 ## Installation
 
-The IBM Watson Swift SDK can be installed with [Cocoapods](http://cocoapods.org/), [Carthage](https://github.com/Carthage/Carthage), or [Swift Package Manager](https://swift.org/package-manager/).
+The IBM Watson Swift SDK can be installed with [Swift Package Manager](https://swift.org/package-manager/), [Cocoapods](http://cocoapods.org/), or [Carthage](https://github.com/Carthage/Carthage).
+
+### Swift Package Manager
+New in version 4.1, the Watson Developer Cloud Swift SDK now supports all services through Swift Package Manager.
```

Comment:
This references version `4.1`, but so far the highest commit level is a `fix`. This means that the semantic release is likely to resolve to version `4.0.2`.

Reply:
My bad I thought a fix resolved to a minor version update